### PR TITLE
Aligned tests with how Chrome v139 changed escaping `<` and `>` in attributes.

### PR DESCRIPTION
**Pull request by pszczesniak**
### 🚀 Summary

*Aligned tests with how Chrome in version 139 changed escaping of `<` and `>` in attributes.*

---

### 📌 Related issues

* Closes #000

---

### 💡 Additional information

*Optional: Notes on decisions, edge cases, or anything helpful for reviewers.*

### CODE REVIEW

**Review by Mati365 — APPROVED** [review]

**Review by Mati365 — APPROVED** [review]

**Review by Mati365 — DISMISSED** [review]

File: packages/ckeditor5-bookmark/tests/manual/bookmark-with-output.js [hunk 1]
```diff
@@ -45,9 +45,7 @@ ClassicEditor
 
 		const iframe = document.querySelector( '#iframe' );
 
-		iframe.contentWindow.document.open();
-		iframe.contentWindow.document.write( editor.getData() );
-		iframe.contentWindow.document.close();
+		iframe.srcdoc = `<!doctype html>${ editor.getData() }`;
 
 		editor.model.document.on( 'change', () => {
 			iframe.contentWindow.document.open();
```
Comment (filipsobol) [inline packages/ckeditor5-bookmark/tests/manual/bookmark-with-output.js]: This probably needs fixing too?

Comment (pszczesniak) [reply packages/ckeditor5-bookmark/tests/manual/bookmark-with-output.js]: Don't know if it need to. It's a proper way, but maybe lets have this code unified. Will push the fix right away.

Comment (pszczesniak) [reply packages/ckeditor5-bookmark/tests/manual/bookmark-with-output.js]: Addressed in https://github.com/ckeditor/ckeditor5/commit/e475f0d89b6aaa1b7387752c509dca328ea78b2d
